mlflow/R/mlflow/Dockerfile.dev

```diff
@@ -3,7 +3,7 @@ FROM rocker/r-ver:4.2.1
 
 WORKDIR /mlflow/mlflow/R/mlflow
 RUN apt-get update -y
-RUN apt-get install git wget libxml2-dev libgit2-dev -y
+RUN apt-get install lsb-release git wget libxml2-dev libgit2-dev -y
```

Comment:
`lsb-release` is called here:

https://github.com/mlflow/mlflow/blob/f490387304c164ad0922578e660de6a4f47f6e3a/mlflow/R/mlflow/.Rprofile#L3-L7

Reply:
This is the changed to run the frequency check properly.